src/components/home-page/_home-page.scss

```diff
@@ -0,0 +1,6 @@
+.home-page {
+  display: flex;
```

Comment:
name file with ".module.scss" to use css modules

Reply:
I don't know anything about css modules. Why should I use them here?